spec/Syncer/InvoiceNinja/ClientSpec.php

```diff
@@ -5,7 +5,7 @@
 use JMS\Serializer\SerializerInterface;
 use Psr\Http\Message\ResponseInterface;
 use Syncer\Dto\InvoiceNinja\Task;
-use Syncer\InvoiceNinja\Client;
+use Syncer\InvoiceNinja\InvoiceNinjaClient as Client;
```

Comment:
By changing the `Syncer\InvoiceNinja\Client`to `Syncer\InvoiceNinja\InvoiceNinjaClient` this spec class fails for two tests. The class `spec\Syncer\InvoiceNinja\ClientSpec` should be renamed to `spec\Syncer\InvoiceNinja\InvoiceNinjaClientSpec`

Reply:
Okay, will do.